src/cpr-mask.helpers.js

```diff
@@ -40,6 +40,8 @@ export function handleMasks(input) {
 	if (mask.pattern !== this.state.maskPattern) {
 		//Mask a new newMask
 		newMask = valueToMask(newValue, mask.pattern, this.props.filler);
+		//Use new pattern to get newValue
+		newValue = maskToValue(newMask, mask.pattern, this.props.filler);
```

Comment:
When we need to update the newMask, we also need to update the newValue. This allows us to paste the value "12345-6789" into the zip code field in end-user-forms-ui when the previous value was "12345"